mkdocs.yml

```diff
@@ -90,6 +90,8 @@ nav:
               - Scanning: docs/kubernetes/cli/scanning.md
           - Operator:
               - Overview: docs/kubernetes/operator/index.md
+      - Compliance:
```

Comment:
@chen-keinan wouldn't this be better under the `Kubernetes` section? It is very specific to k8s.

Reply:
+1

Reply:
@knqyf263 @josedonizetti but we will have to move it later as we are planning to add `aws cis compliance spec` as well so kubernetes  section will match.

Reply:
I think we can have NSA under Kubernetes and CIS under AWS respectively. `trivy k8s` users may not be interested in AWS CIS. We can also have a dedicated page for compliance, but the page just contains links to each page.